proto/src/determined/api/v1/auth.proto

```diff
@@ -15,6 +15,8 @@ message LoginRequest {
   string username = 1;
   // The password of the user.
   string password = 2;
+  // Indicate whether the password is pre-salted & hashed or not.
```

Comment:
```suggestion
  // Indicate whether the provided password is pre-salted & hashed or not.
```
A little less ambiguous.

Reply:
I shoulda committed through the ui 🤦🏼

Reply:
@hamidzr changed this to make sure people know we don't just store the prehashed password ha.